src/Form/FormAnalyzer.js

```diff
@@ -40,8 +41,8 @@ class FormAnalyzer {
          */
         this.isHybrid = false
 
-        // Avoid autofill on our signup page
-        if (window.location.href.match(/^https:\/\/(.+\.)?duckduckgo\.com\/email\/choose-address/i)) {
+        // Avoid autofill on Email Protection web app
+        if (config.isDDGDomain) {
```

Comment:
Using the config to prevent autofill from applying anywhere on the web app

Reply:
Is it worth passing the whole config just for this flag? No strong opinions, just questioning.

Reply:
@GioSensation It feels cleaner to pass the config object about and let the constructor implementation decide what to use from it.

Reply:
Aren't we doing this in the [Scanner](https://github.com/duckduckgo/duckduckgo-autofill/pull/243/files#diff-4f726433357e0b6ad4b671a96eff29a845bd2ccdb7083acde729d06a7728ac04R118) now?

Reply:
@GioSensation Good question, I'll check if there's a reason both are needed